InteractionBase/test/SimpleTest.cpp

```diff
@@ -73,12 +73,11 @@ TEST(InteractionBasePlugin, TextSelect)
 	third->set("Some independent text");
 	manager->endModification();
 
-	auto top = new RootItem(list);
 	auto scene = VisualizationManager::instance().mainScene();
-	scene->addTopLevelItem( top );
+	scene->addTopLevelNode( list );
 	QApplication::processEvents();
 
-	VList* list2 = dynamic_cast<VList*> (top->wrappedItem());
+	VList* list2 = dynamic_cast<VList*> (list);
```

Comment:
This is never going to work. list is a Node and here we're casting to something of type item.

You need to provide a way to get to the item. You can for example get the current viewItem from the scene and then use the findVisualizationOf method to find the corresponding item to the list Node.